libs/interpret-vision/src/lib/VisionExplanationDashboard/VisionExplanationDashboardStyles.ts

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  IStyle,
+  mergeStyleSets,
+  IProcessedStyleSet,
+  getTheme
+} from "@fluentui/react";
+export interface IVisionExplanationDashboardStyles {
+  cohortDropdown: IStyle;
+  cohortPickerLabel: IStyle;
+  cohortPickerLabelWrapper: IStyle;
+  line: IStyle;
+  page: IStyle;
+  searchBox: IStyle;
+  filterButton: IStyle;
+  toolBarContainer: IStyle;
+  mainContainer: IStyle;
+  halfContainer: IStyle;
+  imageListContainer: IStyle;
+  slider: IStyle;
+  tabs: IStyle;
+}
+
+export const visionExplanationDashboardStyles: () => IProcessedStyleSet<IVisionExplanationDashboardStyles> =
+  () => {
+    const theme = getTheme();
+    return mergeStyleSets<IVisionExplanationDashboardStyles>({
+      cohortDropdown: {
+        width: "300px"
+      },
+      cohortPickerLabel: {
+        fontWeight: "600"
+      },
+      cohortPickerLabelWrapper: {
+        paddingBottom: "5px"
+      },
+      filterButton: {
+        height: "32px"
+      },
+      halfContainer: {
+        height: "100%",
+        width: "50%"
+      },
+      imageListContainer: {
+        border: "1px solid #D2D4D6",
+        height: "100%",
+        overflow: "scroll"
+      },
+      line: {
+        borderTop: "1px solid #EDEBE9"
+      },
+      mainContainer: {
+        alignItems: "center",
+        justifyContent: "space-between",
+        width: "100%"
+      },
+      page: {
+        color: theme.semanticColors.bodyText,
+        height: "100%",
+        padding: "0 40px 10px 40px",
```

Comment:
I thought you can specify padding in the tokens on the Stack?
I think I see it in "Vertical Stack - Gap and Padding Sizes" section of this page:
https://developer.microsoft.com/en-us/fluentui#/controls/web/stack
Eg there is this example:
```
const customSpacingStackTokens: IStackTokens = {
  childrenGap: '20%',
  padding: 'm 40px',
};
```

Reply:
If you are resolving comments without change, can you please add reason on why so like are you planning to resolve in coming PR or is it not possible to do this change, etc.?

Reply:
If you are resolving comment without change, can you please add comment on why before resolving? like are you planning to resolve in coming PR or is it not possible to change etc.

Reply:
sorry, forgot to send out the commit with the fixes! Everything I resolved without comment should be fixed within the new commit I'll send out shortly.